openapi/components/schemas/GatewayAccountConfig/Ezeebill.yaml

```
@@ -0,0 +1,42 @@
+description: Ezeebill gateway configuration.
+allOf:
+  - $ref: ../GatewayAccount.yaml
+  - type: object
+    required:
+      - credentials
+    properties:
+      credentials:
+        type: object
+        description: Ezeebill credentials object.
+        properties:
+          merchantId:
+            type: string
+            description: MID in the Ezeebill.
+          accessId:
+            type: string
+            description: Access ID in the Ezeebill.
+            format: password
+            writeOnly: true
+          terminalId:
+            type: string
+            description: Terminal ID in the Ezeebill.
+          operatorId:
+            type: string
+            description: Operator ID in the Ezeebill.
+          password:
+            type: string
+            description: API Access Operator password in Ezeebill.
+            format: password
+            writeOnly: true
+          hashKey:
+            type: string
+            description: Hash Key in Ezeebill.
```

nitpick: I think we should try to use complete sentences here
```suggestion
            description: Access ID of the Ezeebill gateway.
            format: password
            writeOnly: true
          terminalId:
            type: string
            description: Terminal ID of the Ezeebill gateway.
          operatorId:
            type: string
            description: Operator ID of the Ezeebill gateway.
          password:
            type: string
            description: API access operator password of the Ezeebill gateway.
            format: password
            writeOnly: true
          hashKey:
            type: string
            description: Hash key of the Ezeebill gateway.
```